backend/views/views.py

```diff
@@ -63,17 +61,17 @@ def user_page():
                 if len(one_string) < MIN_SEARCH_STR:
                     continue
                 search_parameter = '%{}%'.format(one_string)
-                name_search = User.name.like(search_parameter)
-                alias_search = User.alias.like(search_parameter)
-                email_search = User.email.like(search_parameter)
+                search_like[0] = User.name.like(search_parameter)
```

Comment:
define variable search_like

Reply:
fixed

Reply:
failed with: IndexError: list assignment index out of range
Please run code on your local machine first

Reply:
fexed